src/handlers/timeouts/handler.js

```diff
@@ -473,5 +477,6 @@ module.exports = {
   registerAllHandlers,
   registerTimeoutHandler,
   isRunning,
-  stop
+  stop,
+  running
```

Comment:
This is not useful for anything, it just adds a property to the exports object, initialized with the default value of false.